easybuild/easyconfigs/l/libdwarf/libdwarf-20140805-GCC-4.9.2.eb

```diff
@@ -0,0 +1,35 @@
+easyblock = 'MakeCp'
+
+name = 'libdwarf'
+version = '20140805'
+
+homepage = 'http://www.prevanders.net/dwarf.html'
+description = """The DWARF Debugging Information Format is of interest to programmers working on compilers
+and debuggers (and anyone interested in reading or writing DWARF information))"""
+
+toolchain = {'name': 'GCC', 'version': '4.9.2'}
+
+sources = [SOURCE_TAR_GZ]
+source_urls = ['http://www.prevanders.net']
+
+dependencies = [('libelf', '0.8.13')]
+
+with_configure = True
+preconfigopts = 'CFLAGS="-fPIC $CFLAGS" '
+configopts = "--enable-shared "
+
+# This is dirty but libdwarf cannot find it's own library in the build process...
+prebuildopts = ' LD_LIBRARY_PATH="../libdwarf:$LD_LIBRARY_PATH" '
+
+files_to_copy = [
+    (["dwarfdump2/dwarfdump"], "bin"),
+    (["libdwarf/libdwarf.a", "libdwarf/libdwarf.so"], "lib"),
+    (["libdwarf/libdwarf.h", "libdwarf/dwarf.h"], "include"),
+]
+
+sanity_check_paths = {
+    'files': [],
+    'dirs': ["bin", "lib", "include"]
```

Comment:
Because if the files failed to copy, there would already have been an error before...
Now, i'm just confirming that the dirs are there with files in them.